tools/ci_build/github/azure-pipelines/templates/py-packaging-stage.yml

```diff
@@ -144,11 +144,11 @@ stages:
 
 
   - ${{ if eq(parameters.enable_linux_gpu, true) }}:
-    - job: Linux_py_GPU_Wheels
-      timeoutInMinutes: 120
+    - job: Linux_py_Wheels_openmp
```

Comment:
Is this change correct? It looks like the condition is still based on `enable_linux_gpu`, but the changes below seem to switch to a CPU build.

Reply:
Moving doc generation to a separate PR